src/AwsClient.php

```diff
@@ -554,7 +554,11 @@ private function setClientBuiltIns($args)
         $config = $this->getConfig();
         $service = $args['service'];
 
-        $builtIns['SDK::Endpoint'] = isset($args['endpoint']) ? $args['endpoint'] : null;
+        $builtIns['SDK::Endpoint'] = isset($config['endpoint_url'])
```

Comment:
Should we merge this soon so we can add this to the deprecation with new syntax?  Like `$config['endpoint_url'] ?? $args['endpoint'] ?? null;`